ui/plugins/pinboard.js

```diff
@@ -101,6 +101,15 @@ treeherder.controller('PinboardCtrl', [
             }
         };
 
+        $scope.retriggerAllPinnedJobs = function() {
+            var jobs = [];
+            // pushing pinned jobs to a list.
+            for (var job in $scope.pinnedJobs) {
+                jobs.push($scope.pinnedJobs[job]);
+            }
+            $scope.retriggerJob(jobs);
```

Comment:
@maurodoglio 
$scope.pinnedJobs is a dictionary of jobs: https://github.com/mozilla/treeherder/blob/1900b665e4f102490ff2b8cd6d49ef6c3074f6ad/ui/js/services/pinboard.js#L16
But we want to send an array of jobs, so I am extracting the job in an array and sending it. Is there a better way?